plugins/resource-monitor-plugin/src/resource-monitor-plugin.ts

```diff
@@ -193,6 +185,6 @@ export class ResMon {
 }
 
 export async function getNamespace(): Promise<string> {
-  const devfile = await che.devfile.get();
-  return devfile.metadata?.attributes ? devfile.metadata.attributes.infrastructureNamespace : '';
+  const workspace = await che.workspace.getCurrentWorkspace();
+  return workspace.attributes?.['infrastructureNamespace'] || workspace.namespace || '';
```

Comment:
hi, `workspace.attributes` is only a che-server implementation (so we should keep devfileService to grab the attributes information)

Reply:
@benoitf I've tried to get namespace from devfileService and `devfile.metadata?.attributes` returns
```json
{"metadata-name-field":"name"}
```
(no `infrastructureNamespace` here)
but if use workspaceService, `workspace.attributes?.['infrastructureNamespace']` returns real namespace: `admin-che`

Reply:
maybe there is a bug on how we transmit properties into devfile service but we can't rely on workspace.attributes field when using devWorkspace so we shouldn't introduce this dependency that was removed

Reply:
Fixed. Now `infrastructureNamespace` attribute is reading from devfle service.